auto_cpufreq/bin/auto_cpufreq.py

```diff
@@ -115,7 +88,7 @@ def config_info_dialog():
             battery_setup()
             battery_get_thresholds()
             conf.notifier.start()
-            if os.getenv("PKG_MARKER") == "SNAP":
+            if IS_INSTALLED_WITH_SNAP:
```

Comment:
I also like what you did with global vars, as indeed this might be more readable.